docs/user/appendix/contribution-guidelines.md

```diff
@@ -170,6 +170,46 @@ And this is how those tabs will look like:
 
 {{#include ../../shared/tab_footer.md}}
 
+#### Tab tooltips
+
+When a tab is disabled i.e. does not have any content, it is possible to provide extra context
```

Comment:
When creating new content, what should we do? Just fill in for the existing tabs we know of or add them all and add a tooltip? May be worth making a call here (whatever that is) for consistency moving forward.

Reply:
There are guidelines about that in [this section](https://mozilla.github.io/glean/book/appendix/contribution-guidelines.html#tabs) of the contribution guidelines:

> Every tab section contains tabs for each Glean language binding, even that language binding does not provide the API in question. In this case, the tab div is still there without any inner HTML. When that is the case that tab will be rendered in a disabled state.

BUT I agree that this is not very clear 😄 

The idea is that tabs sections always have all the tabs, regardless if a given language binding doesn't provide the API being exemplified. When that is the case the tab should be added with no content inside and `tabs.js` will do the job of making sure emtpy tabs are shown in a disabled state.

By default, if you don't provide _any_ tooltip text, the default message is `Javascript does not provide this API`, if you provide a `data-bug` it shows `Javascript does not provide this API. Follow Bug XXXXX for updates.` and if you provide a `data-info` it will show whatever text you give it. The tooltip text itself depends on whether you think the default message is enough or not.